### DISCUSSION TITLE
Where can COLOR SCHEME be used?

### DISCUSSION THREAD

**Discussion by HeveraletLaidCenx** [darkreader/darkreader#8616] [Q&A]
I look through the [CONTRIBUTING.md](https://github.com/darkreader/darkreader/blob/master/CONTRIBUTING.md), and I'm interested in the section about [**color scheme**](https://github.com/darkreader/darkreader/blob/master/CONTRIBUTING.md#adding-a-new-color-scheme).

It is really detailed, I think I got that how to add a new color scheme to the config and then PR, but... I just have no idea where it can be used or called from the plugin.

I checked the [help page](https://darkreader.org/help/en/), the other sections of the [CONTRIBUTING.md](https://github.com/darkreader/darkreader/blob/master/CONTRIBUTING.md), the [IS](https://github.com/darkreader/darkreader/issues?q=color+scheme) and the [discussion](https://github.com/darkreader/darkreader/discussions?discussions_q=color+scheme), but I am still confused, seems I can't findout a brief intro of how to use color scheme... Is it used in one of the render mode or somewhere else? Or am I missing some relevant instructions should be easily seen? Help plllllz!

**Comment by Gusted** [comment]
https://github.com/darkreader/darkreader/discussions/6707#discussioncomment-1295216 Checkout the new UI.
